### DISCUSSION TITLE
CityGML coordinates understanding problem

### DISCUSSION THREAD

**Discussion by triin1111** [cityjson/specs#174] [Q&A] [answered]
Hi! 
I have a problem, if anyone is maybe able to help. I have imported data from CityGML into PostGre database. In this data, the wall textures of buildings are isometric, but I need to make them straight. I can translate them from <app:textureCoordinates ring> to actual wall coordinates, but for that I need to know the starting coordinate of <app:textureCoordinates ring>, which coordinate index corresponds to lower left corner of the wall. 
Is there information about that in the CityGML database or should I determine it myself somehow? 

Thank you so much for help!

**Comment by balazsdukai** [comment]
It seems to me that you are using 3dcitydb, is that correct?

**Reply by triin1111** [reply]
Yes, that's correct.

**Reply by balazsdukai** [reply, marked as answer]
In that case, it's better if you ask the question in the 3dcitydb repository https://github.com/3dcitydb/3dcitydb . It has its own data model and we might not be able to help you here.

**Reply by triin1111** [reply]
Will do, thanks for your help!
